index.html

```diff
@@ -10,16 +11,16 @@
     <aside>
       <h2>A Side Bar</h2>
     </aside>
-    <div class='main'>
+    <main class='main'>
```

Comment:
:+1: and I would probably remove the main class at this point, unless it's being used for something in particular.